database/migrations/2020_11_04_160917_create_loan_requests_table.php

```diff
@@ -19,7 +19,7 @@ public function up()
             $table->foreignId('user_id')->constrained('users');
             $table->foreignId('status_changed_by')->constrained('users');
             
-            $table->string('status')->default('pending');
+            $table->string('status')->default(LoanRequest::DEFAULT_STATUS);
```

Comment:
Rather than using a property `DEFAULT_STATUS` use the exact status which is the default. In that way, it's more readable.
Explained the reason above in the `users` table comment.

Reply:
Fix it in other places too.